packages/fxa-settings/src/pages/Signin/interfaces.ts

```diff
@@ -197,6 +197,7 @@ export interface NavigationOptions {
     // keyFetchToken is included if options.keys=true
     // This (and unwrapBKey) will never exist for the cached signin (prompt=none)
     keyFetchToken?: hexstring;
+    sessionVerified?: boolean;
```

Comment:
I wonder if this should be optional? Making it required might ensure we don't miss another spot were it should actually be getting set.